### PR TITLE
[Documentation][CookBook] Specific SyliusBundles extension

### PR DESCRIPTION
**Pull request by Zales0123**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

Rebased and slightly improved https://github.com/Sylius/Sylius/pull/8517, thank you @Adraesh for your tremendous work!

### CODE REVIEW

**Comment by Zales0123** [comment]
ping @CoderMaggie

File: docs/cookbook/configuration/extending-bundles-other-em.rst [hunk 1]
```diff
@@ -0,0 +1,738 @@
...
+         *
+         * @return Address
+         */
+        public function getDefaultAddress()
+        {
+            return $this->defaultAddress;
+        }
+
+        /**
+         * Set user
```
Comment (lchrusciel) [inline docs/cookbook/configuration/extending-bundles-other-em.rst]: Do we need all these comments? Doc will be more compact without it

**Comment by lchrusciel** [comment]
Thanks Mateusz! 🐋

**Comment by lchrusciel** [comment]
Forgive me my ignorance, and Thomas as well! 🐋 ☮️

**Pull request merged**